static/js/common.js

```diff
@@ -109,15 +109,15 @@ exports.adjust_mac_shortcuts = function (key_elem_class, require_cmd_style) {
         return;
     }
 
-    var keys_map = new Map([
-        ['Backspace', 'Delete'],
-        ['Enter', 'Return'],
-        ['Home', 'Fn + ←'],
-        ['End', 'Fn + →'],
-        ['PgUp', 'Fn + ↑'],
-        ['PgDn', 'Fn + ↓'],
-        ['Ctrl', '⌘'],
-    ]);
+    var keys_map = {
```

Comment:
Alternatively, maybe we should whitelist `Map` and `Set`, since IE supports them.